samples/subsys/usb/uvc/prj.conf

```diff
@@ -0,0 +1,13 @@
+CONFIG_LOG=y
+CONFIG_VIDEO=y
+CONFIG_VIDEO_BUFFER_POOL_NUM_MAX=4
+CONFIG_VIDEO_BUFFER_POOL_SZ_MAX=7300
+CONFIG_VIDEO_EMUL_IMAGER=y
+CONFIG_VIDEO_EMUL_RX=y
+CONFIG_VIDEO_LOG_LEVEL_DBG=y
+CONFIG_USB_DEVICE_STACK=n
+CONFIG_USB_DEVICE_STACK_NEXT=y
+CONFIG_USBD_VIDEO_CLASS=y
+CONFIG_USBD_VIDEO_LOG_LEVEL_DBG=y
+CONFIG_SAMPLE_USBD_PID=0x0010
+CONFIG_SAMPLE_USBD_PRODUCT="UVC sample"
```

Comment:
+CONFIG_USBD_LOG_LEVEL_WRN=y
+CONFIG_UDC_DRIVER_LOG_LEVEL_WRN=y

Reply:
Done! This too?
```diff
-CONFIG_USBD_VIDEO_LOG_LEVEL_DBG=y
+CONFIG_USBD_VIDEO_LOG_LEVEL_WRN=y
```